<ui version="4.0" >
 <class>ModeMenu</class>
 <widget class="QDialog" name="ModeMenu" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>350</width>
    <height>250</height>
   </size>
  </property>
  <property name="baseSize" >
   <size>
    <width>350</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Change Rx Mode</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layout3" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>190</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QPushButton" name="buttonOk" >
      <property name="minimumSize" >
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="text" >
       <string>&amp;OK</string>
      </property>
      <property name="shortcut" >
       <string/>
      </property>
      <property name="autoDefault" >
       <bool>true</bool>
      </property>
      <property name="default" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>110</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="buttonCancel" >
      <property name="minimumSize" >
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="text" >
       <string>&amp;Cancel</string>
      </property>
      <property name="shortcut" >
       <string/>
      </property>
      <property name="autoDefault" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="Reverse" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>30</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Reverse</string>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="ParityLayout" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>90</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title" >
    <string>Parity</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignHCenter</set>
   </property>
   <widget class="QRadioButton" name="Odd" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Odd</string>
    </property>
    <property name="buttonGroupId" stdset="0" >
     <number>1</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="Even" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>60</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Even</string>
    </property>
    <property name="buttonGroupId" stdset="0" >
     <number>2</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="None" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>None</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
    <property name="buttonGroupId" stdset="0" >
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="RxMode" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="Spacing" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>19</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title" >
    <string>RTTY Spacing</string>
   </property>
   <widget class="QSpinBox" name="RTTYSpacing" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="suffix" >
     <string> Hz</string>
    </property>
    <property name="minimum" >
     <number>0</number>
    </property>
    <property name="maximum" >
     <number>999</number>
    </property>
    <property name="value" >
     <number>170</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="StopbitsLayout" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>90</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title" >
    <string>Stopbits</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignHCenter</set>
   </property>
   <widget class="QRadioButton" name="One" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>1</string>
    </property>
    <property name="buttonGroupId" stdset="0" >
     <number>0</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="Onepoint5" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>1.5</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
    <property name="buttonGroupId" stdset="0" >
     <number>1</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="Two" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>2</string>
    </property>
    <property name="buttonGroupId" stdset="0" >
     <number>2</number>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <includes>
  <include location="local" >constants.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>ModeMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>ModeMenu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
